src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTests.cs

```diff
@@ -1405,6 +1404,18 @@ public void M()
             await AssertFormatAfterTypeCharAsync(code, expected);
         }
 
+        [WorkItem(11642, "https://github.com/dotnet/roslyn/issues/11642")]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task FormatArbitraryNodeParenthesizedLambdaExpression()
+        {
+            // code equivalent to an expression synthesized like so:
```

Comment:
Not sure why the comment is here. The test seems to stand by itself quite nicely.